integrations/tensorflow/test/python/lit.local.cfg

```diff
@@ -0,0 +1 @@
+config.suffixes = []
```

Comment:
Is this the best way to tell lit to not look at a directory?

Reply:
For now. Will rework in a followup.